### DISCUSSION TITLE
EMail to info@revsemi.com bounced

### DISCUSSION THREAD

**Discussion by RossAddinall** [eskiyerli/revedaRelease#15] [General]
Hi,

A real quick one - I sent an email to info@revsemi.com as shown on your website and it bounced with:
```
Address not found
--
Your message wasn't delivered to info@revsemi.com because the address couldn't be found, or is unable to receive mail.

```

Regards,

Ross

**Comment by eskiyerli** [comment]
Thanks for letting me know. Email providers were changed recently. I must have forgotten to add it. I will do that. Thanks.

**Comment by eskiyerli** [comment]
Just added it and checked it. You could send an email if you wish to.
